### DISCUSSION TITLE
Pydantic 2.4.2 tag created but not published on PyPi

### DISCUSSION THREAD

**Discussion by vincentsarago** [pydantic/pydantic#7685] [Question]
sorry for opening a discussion for this but couldn't find another way 🤷‍♂️ 

Tag 2.4.2 was created and pushed on GitHub yesterday but package wasn't released on pypi. It seems that https://github.com/pydantic/pydantic/actions/runs/6329478197/job/17190061782 needs approval 🤷 

Thank you

cc @dmontagu

**Comment by hramezani** [comment]
Yes, you are right. I will be published most probably today!

**Comment by samuelcolvin** [comment]
It should release now, no one approved the release, hence the wait.

**Reply by vincentsarago** [reply]
Thank you @samuelcolvin 💪 

really appreciate 🙏
